governance/votes/Executive vote - November 2, 2022.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Template - [Executive Vote] Maker Teleport Deployment on Starknet, Offboarded Core Units Budgets and Streams, Oracle Whitelist Additions, and BlockTower SPF Funding - November 02, 2022
+summary: Maker Teleport Technical Deployment on Starknet, Golden Parachute MKR and DAI budget transfers for EVENTS-001, SH-001, and RWF-001, yank MKR and DAI budget streams for EVENTS-001, SH-001, and RWF-001, whitelist Oasis.app on rETH/USD oracle, and BlockTower Legal Work and Risk Work SPF Funding. 
+date: 2022-11-02T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Maker Teleport Deployment on Starknet, Offboarded Core Units Budgets and Streams, Oracle Whitelist Additions, and BlockTower SPF Funding - November 02, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The Maker Teleport Global [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased by **1 million DAI** to **3 million DAI**.
+- A total of **210,998 DAI** will be transferred to multiple offboarded Core Units as part of their respective Golden Parachutes.
+- A total of **169.5 MKR** will be transferred to multiple offboarded Core Units as part of their respective Golden Parachutes.
+- DAI streams for various now offboarded will be yanked.
+- The MKR stream for the now offboarded [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) will be yanked.
+- [Oasis.app](https://oasis.app) will be whitelisted on the rETH/USD oracle.
+- **258,000 DAI** will be transferred to the BlockTower Legal and Risk Work SPF multi-sig as a result of the successful [Special Purpose Fund (SPF) application](https://vote.makerdao.com/polling/QmcV2pMZ#vote-breakdown).
```

Comment:
1. try and keep things human readable
2. this is a summary, we will add more details below

```suggestion
- A total of **210,998 DAI**, itemized below, will be transferred to two offboarded Core Units as part of their respective Golden Parachutes.
- A total of **169.5 MKR**, itemized below, will be transferred to two offboarded Core Units as part of their respective Golden Parachutes.
- DAI streams to three now offboarded Core Units, itemized below, will be cancelled.
- The MKR stream to the now offboarded [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) will be cancelled, as detailed below.
- [Oasis.app](https://oasis.app) will be whitelisted on the rETH/USD oracle, as detailed below.
- **258,000 DAI** will be transferred to the BlockTower Legal and Risk Work SPF multi-sig as a result of the successful [Special Purpose Fund (SPF) application](https://vote.makerdao.com/polling/QmcV2pMZ#vote-breakdown), as detailed below.
```